packages/ui-core/src/components/slider/interval-slider.stories.tsx

```diff
@@ -82,3 +83,94 @@ withRuler.story = {
     },
   },
 };
+
+const themingIntervals = [{ minimum: 0, maximum: 35, step: 5 }];
+
+const generateRulerSettings = ({ minimum, maximum, step }) => {
+  const middlePoints = [];
+  const numberOfSteps = Math.floor((maximum - minimum) / step);
+
+  if (numberOfSteps >= 1) {
+    for (let i = 1; i <= numberOfSteps; i++) {
+      middlePoints.push({
+        label: `${minimum + i * step}`,
+        position: `${(100 * (minimum + i * step)) / maximum}%`,
+      });
+    }
+  }
+
+  return [
+    {
+      label: minimum,
+      position: '0%',
+    },
+    ...middlePoints,
+    {
+      label: maximum,
+      position: '100%',
+    },
+  ];
+};
+
+const LabelWrapper = ({ children }) => (
+  <div style={{ marginTop: 10 }}>{children}</div>
+);
+
+export const withRulerTheming = () => {
```

Comment:
Based on what we discussed the user should be able to use values between defined ruler ranges.
In use case on the image - it should be **12**
<img width="353" alt="Zrzut ekranu 2021-05-26 o 07 57 10" src="https://user-images.githubusercontent.com/7753369/119609567-195d0e00-bdf8-11eb-96a3-fda10bd47bf9.png">